src/app/Services/API/Requests/index.ts

```diff
@@ -56,7 +56,9 @@ export class RequestApi extends Http {
       }
       return this.instance
         .get<any>(
-          'https://arpav.geobeyond.dev/api/v2/coverages/forecast-data?',
+          //'https://arpav.geobeyond.dev/api/v2/coverages/forecast-data?',
```

Comment:
I would get rid of the commented out values throughout, as they are not needed anymore with this change